Macbook-setup.md

```diff
@@ -0,0 +1,163 @@
+_Note: this will also install the XCode command line tools._
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+---
+## Install Python 3.6.9 via pyenv
+
+At time of writing, MacOS ships with Python 2.7.16 (/usr/bin/python) and Python 3.8.2 (/usr/bin/python3). We will leave these installed but will not use them, instead opting for pyenv to manage python versions (starting with Python 3.6.9 to match current production server).
+```
+brew install zlib sqlite bzip2 libiconv libzip xz pyenv 
+LDFLAGS="-L$(brew --prefix zlib)/lib -L$(brew --prefix bzip2)/lib" CPPFLAGS="-I$(brew --prefix xz)/include" pyenv install --patch 3.6.9 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch\?full_index\=1)
+```
+
+Add the following line to ~/.zshrc:
+```
+export PATH=/Users/<your system username>/.pyenv/versions/3.6.9/bin:$PATH
+```
+Then restart terminal. Now `python --version` and `python3 --version` should show version 3.6.9.
+
+Next install pipenv and envdir:
+```
+pip install pipenv envdir
+```
+
+---
+## Install updated version of bash
+
+At time of writing, MacOS ships with bash version 3.x. A newer version is requires to run some of our bash scripts, so we install bash 5.x with Homebrew:
+```
+brew install bash
+```
+Close are restart terminal and check bash version 5.x is working. (Note: Homebrew installs bash 5.x to /usr/local/bin/bash. bash 3.x is still installed at /bin/bash)
+```
+bash --version
+```
+
+---
+## Create a PostgreSQL 10 instance using Postgres.app
+
+1. Launch Postgres from Applications folder. 
+2. Click the show panel button in the bottom left corner.
+3. Click the + button to add a new PostgreSQL instance.
+4. Name the new instance and select version 10 from the dropdown. Leave port as 5432.
+5. Press 'Create Server' button.
+
+You should now have a PostgreSQL server running with these default settings:
+- Host: localhost
+- Port: 5432
+- User: _your system username_
+- Database: _same as user_
+- Password: _none_
+
+## Add Postgres 10 binary folder to PATH
+
+Add the following line to ~/.zshrc:
+```
+export PATH=$PATH:/Applications/Postgres.app/Contents/versions/10/bin
+```
+Then close and restart terminal. You should now be able to run psql from anywhere.
+```
+psql --version
+```
+
+---
+## Clone git repos
+
+Create a workspace (e.g. ~/workspace/seattleflu) and clone the three primary repos:
+```
+mkdir ~/workspace
+mkdir ~/workspace/seattleflu
+cd ~/workspace/seattleflu
+git clone https://github.com/seattleflu/id3c
+git clone https://github.com/seattleflu/id3c-customizations
+git clone https://github.com/seattleflu/backoffice
+```
+
+Make sure you can run `pipenv sync` in the id3c and id3c-customizations folders:
+```
+cd ~/workspace/seattleflu/id3c
+pipenv sync
+pipenv run pytest -v
+pipenv run id3c --help
+```
+
+```
+cd ~/workspace/seattleflu/id3c-customizations
+pipenv sync
+pipenv run pytest -v
+```
+
+---
+## Create folder for environment variables
+
```

Comment:
It may be useful to add in a little explainer here in addition to the example set up provide, something like:

```
We use envdir for managing environment variables. You can set up an env.d directory locally. It will contain individual files for every environment variable, and subdirectories to further organize. You can see one example for the organizational structure is the backoffice/id3c-production/env.d. 

By convention, we store Redcap API tokens are stored in file names using this pattern: REDCAP_API_TOKEN_redcap.iths.org_{PROJECT_ID}
```

Reply:
Updated to match your suggestion.